src/ray/common/asio/asio_chaos.cc

```diff
@@ -121,17 +121,17 @@ class DelayManager {
   std::pair<int64_t, int64_t> default_delay_range_us_ = {0, 0};
 };
 
-static DelayManager _delay_manager;
+DelayManager delay_manager;
 }  // namespace
 
-int64_t get_delay_us(const std::string &name) {
+int64_t GetDelayUs(const std::string &name) {
   if (RayConfig::instance().testing_asio_delay_us().empty()) {
```

Comment:
Should we avoid using env for testing?
CC @edoakes I had a discussion with you on this.

Reply:
What's your recommendation here?

Reply:
global var

Reply:
or redesign the interface

Reply:
Ok, let's do it in a follow-up? Want to know more details about the discussion.

Reply:
<img width="1052" alt="image" src="https://github.com/user-attachments/assets/70b67ee7-a458-4fcb-927d-eceb2035f8b6" />
<img width="1048" alt="image" src="https://github.com/user-attachments/assets/9ee7d55b-5d3f-4b29-bf70-ccdb8bb7b641" />
<img width="1039" alt="image" src="https://github.com/user-attachments/assets/55f09858-0f1f-424b-a14b-fea67b5e20f1" />